app/assets/javascripts/users/settings/teams/show.js

```diff
@@ -187,6 +187,8 @@
   }
 
   function initTeamSharePermission() {
+    $('.team-share-permission')
+      .css('cursor', `${$('.team-share-permission').attr('disabled') ? 'not-allowed' : 'cursor'}`);
```

Comment:
could the same thing be achieved just via css using the selector `.team-share-permission:disabled`?